pkg/alertmanager/amcfg.go

```diff
@@ -1447,6 +1513,10 @@ func (sc *snsConfig) sanitize(amVersion semver.Version, logger log.Logger) error
 	return sc.HTTPConfig.sanitize(amVersion, logger)
 }
 
+func (tc *telegramConfig) sanitize(amVersion semver.Version, logger log.Logger) error {
+	return tc.HTTPConfig.sanitize(amVersion, logger)
```

Comment:
see my previous comment around verifying we can support this by checking the version.